src/app/qgslayerstylingwidget.cpp

```diff
@@ -812,7 +790,7 @@ bool QgsLayerStyleManagerWidgetFactory::supportsLayer( QgsMapLayer *layer ) cons
       return false;  // TODO
 
     case QgsMapLayerType::PointCloudLayer:
-      return false;  // TODO
```

Comment:
This change isn't used anywhere -- QgsLayerStyleManagerWidgetFactory isn't inherited by any of the point cloud QgsMapLayerConfigWidgetFactory subclasses.

I'd suggest collapsing the point cloud (and vector tile layer) cases into the final plugin/annotation layer case and removing the "TODO" notes, since there's no TODO required here.